docker/bfe-config.js

```diff
@@ -0,0 +1,487 @@
+                    });
+		            //bfestore.n32store(data, url, tempstore, callback);
+                    },
+                error: function(XMLHttpRequest, textStatus, errorThrown) {
+                    bfelog.addMsg(new Error(), "ERROR", "FAILED to load external source: " + url);
+                    bfelog.addMsg(new Error(), "ERROR", "Request status: " + textStatus + "; Error msg: " + errorThrown);
+                }
+            });
+        }
+
+        function retrieveLDS(uri, bfestore, loadtemplates, bfelog, callback){
+
+        var url = config.url + "/profile-edit/server/retrieveLDS";
+
+            $.ajax({
+                dataType: "json",
+                type: "GET",
+                async: false,
+                data: { uri: uri},
+                url: url,
+                success: function (data) {
+                    bfelog.addMsg(new Error(), "INFO", "Fetched external source baseURI" + url);
+                    bfelog.addMsg(new Error(), "DEBUG", "Source data", data);
+                    bfestore.store = bfestore.jsonldcompacted2store(data, function(expanded) {
+                       bfestore.store = [];
+                       tempstore = bfestore.jsonld2store(expanded);
+                       callback(loadtemplates);
+                    });
+                    //bfestore.n32store(data, url, tempstore, callback);
+                    },
+                error: function(XMLHttpRequest, textStatus, errorThrown) {
+                    bfelog.addMsg(new Error(), "ERROR", "FAILED to load external source: " + url);
+                    bfelog.addMsg(new Error(), "ERROR", "Request status: " + textStatus + "; Error msg: " + errorThrown);
+                }
+            });
+        }
+
+
+        function deleteId(id, csrf, bfelog){
+            var url = config.url + "/verso/api/bfs/" + id;
+
+            //$.ajaxSetup({
+            //    beforeSend: function(xhr, settings){getCSRF(xhr, settings, csrf);}
+            //});
+
+            $.ajax({
+                type: "DELETE",
+                url: url,
+                dataType: "json",
+                csrf: csrf,
+                success: function (data) {
+                    bfelog.addMsg(new Error(), "INFO", "Deleted " + id);
+                    },
+                error: function(XMLHttpRequest, textStatus, errorThrown) {
+                    bfelog.addMsg(new Error(), "ERROR", "FAILED to delete: " + url);
+                    bfelog.addMsg(new Error(), "ERROR", "Request status: " + textStatus + "; Error msg: " + errorThrown);
+                }
+            });
+
+        }
+
+        var config = {
+/*            "logging": {
+                "level": "DEBUG",
+                "toConsole": true
+            },*/
+            "url" : "http://editor:8000",
+            "baseURI": "http://ld4p.org/",
+            "basedbURI": "http://ldp4p.org:8000",
+            "resourceURI": "http://ldp4p.org/resources",
+            "profiles": [
```

Comment:
If the repo that docker clones is going to be `https://github.com/sul-dlss/bfe.git` then we may need to do something about the profiles that are loaded: trim this array, and also add the default profiles we will use to `static/profiles/bibframe` (which does not currently exist in this repo).

Reply:
Agree, for now how does having one Audio Recording profile and one Video (maybe the Moving Image BlueRay DVD) profiles. Any suggestions on other profiles we should keep would be great.
  
I think the default profiles that were in `static/profiles/bibframe` are now being loaded from the loc website with a GET call i.e. http://id.loc.gov/ontologies/bibframe/Instance.json etc.